srcpkgs/xinput_calibrator/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'xinput_calibrator'
 pkgname=xinput_calibrator
-version=0.7.5
-revision=3
+version=0.8.0
+revision=1
 build_style=gnu-configure
```

Comment:
switch to meson?

Reply:
Ah, I haven't noticed that [they have added Meson](https://lists.x.org/archives/xorg-announce/2024-July/003522.html).

Reply:
I had to add a patch because of this, but it's working.